client/src/components/CarouselItem.jsx

```diff
@@ -71,14 +71,14 @@ const StarWrapper = styled.div`
 const CarouselItem = (props) => {
     const {item} = props;
     let price;
-    if (!item.onSale) {
+    if (!item.onsale) {
         price = <Price>{item.price}</Price>
     } else {
         price = <OnSale>{item.price}</OnSale>
     }
     return (
         <Item>
-            <Image src={item.imageUrl}/>
+            <Image src={item.imageurl}/>

```

Comment:
I figured 74, and 81 should be camelCase

Reply:
they SHOULD be, but the way they are now allows them to move from database to front end without errors. I will look into changing them in backend so they are legible to front end camelCase